site/content/docs/concepts/pipelines.md

```diff
@@ -12,11 +12,11 @@ Using an automated release tool like CodePipeline helps make your release manage
 
 Copilot can set up a CodePipeline for you with a few commands - but before we jump into that, let's talk a little bit about the structure of the pipeline we'll be generating. Our pipeline will have the following basic structure:
 
-1. __GitHub Source__ - when you push to a configured branch (main by default), a new pipeline execution is triggered.
-2. __Build Stage__ - after your code is pulled from GitHub, your service's container image is built and published to every environment's ECR repository.
-3. __Deploy Stages__ - after your code is built, you can deploy to any and all of your environments, with optional post-deployment tests or manual approvals.
+1. __Source Stage__ - when you push to a configured GitHub or CodeCommit branch ('main' or 'master', respectively, by default), a new pipeline execution is triggered.
+2. __Build Stage__ - after your code is pulled from your provider, your service's container image is built and published to every environment's ECR repository.
```

Comment:
is provider a common term that folks will understand or is it our internal source code lingo?

Reply:
Oh, good question. Looks like "git hosting service," "code repository management system," "source control service," "version control repository hosting service," and variations thereof are all used in addition to "git provider." I was trying to avoid using "GitHub or CodeCommit" over and over, but you're right-- "provider" isn't clear. Changing it to "after your source code is pulled from your repository host" unless people have better ideas!